editor/editor_inspector.cpp

```diff
@@ -901,6 +913,8 @@ void EditorProperty::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_label", "text"), &EditorProperty::set_label);
 	ClassDB::bind_method(D_METHOD("get_label"), &EditorProperty::get_label);
 
+	ClassDB::bind_method(D_METHOD("_set_read_only", "read_only"), &EditorProperty::_set_read_only);
```

Comment:
It should be
`BIND_VMETHOD(MethodInfo("_set_read_only", PropertyInfo(Variant::BOOL, "readonly"))`
and after all methods.